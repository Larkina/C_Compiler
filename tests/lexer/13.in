double i = 12.e.1;