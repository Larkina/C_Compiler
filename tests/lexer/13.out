Token{value=double, text=double, type=KEY_WORD}
Token{value=i, text=i, type=VAR}
Token{value==, text==, type=ASSIGN}
Incorrect token on 1 line, 5 pos: Incorrect float number
